skabase/SKABaseDevice/SKABaseDevice/SKABaseDevice.py

```diff
@@ -360,6 +364,18 @@ def read_storageLoggingLevel(self):
     def write_storageLoggingLevel(self, value):
         # PROTECTED REGION ID(SKABaseDevice.storageLoggingLevel_write) ENABLED START #
         self._storage_logging_level = value
+        # if self._storage_logging_level == 1:
+        #     logger.setLevel(logging.FATAL)
```

Comment:
Not sure why all of this is being removed.  Devices have their own logger instances, so we would like to adjust them.  Here we could use `MODULE_LOGGER`, but better to set `self.logger = MODULE_LOGGER` in `init_device()`, and then use `self.logger` throughout this class.

Reply:
Yeah Anton, you are right. This all should be in Base Device. But I was exploring the Logging feature so added everything in Test Device rather than playing with the Base Device. But now since logging functionality works, I plan to move the code to Base Device to prevent repetition in other devices.

Reply:
Yes, have done that.